README.md

```diff
@@ -11,8 +11,27 @@ The repository for the backend can be found [here](https://github.com/mozilla/fi
 
 ## Prerequisites
 
-- Node
-- [Yarn](https://www.npmjs.com/package/yarn)
+
+## [Node](https://nodejs.org/en/)
+
+* ## Install node:
+
+Binaries, installers, and source tarballs are available at
+<https://nodejs.org/en/download/>.
```

Comment:
It has been a while since I installed Node.  are there any surprises?  I know Node gets installed in two locations: local and global (maybe Yarn did that).  Getting eslint to upgrade last month involved me going into `appdata` and deleting the global (or was that the local?) version.

Also mention the minimum version of node, which is 16